agent/agent-tooling/src/main/java/com/microsoft/applicationinsights/agent/internal/wasbootstrap/configuration/Configuration.java

```diff
@@ -579,6 +584,66 @@ public void validate() throws FriendlyException {
         public GcReportingLevel reportingLevel = GcReportingLevel.TENURED_ONLY;
     }
 
+    public static class AadAuthentication {
```

Comment:
this is through config, each customer will need to provide these config properties? how do they get it?

Reply:
This will be available in the Azure Identity documentation. However I am also thinking to add some documentation on Java agent's docs.

Reply:
is it Azure AD Identity Protection documentation or managed identities for Azure? can you share the link? and document it in this class?